spec/models/category_spec.rb

```diff
@@ -3,4 +3,37 @@
 describe Category do 
 	it { should have_many(:videos)}
 
+	describe "#recent_videos" do 
+		it "returns an empty array if the category has no videos in it" do
+			drama = Category.create(name: "drama")
+			expect(drama.recent_videos).to eq([])
+		end
+
+		it "returns all videos in category if there are less than 6" do
+			drama = Category.create(name: "drama")
+			monk =	Video.create(title: "monk", description: "Sample Video", category: drama, created_at: 1.day.ago )
+			gotham =	Video.create(title: "gotham", description: "Sample Video", category: drama )
+			expect(drama.recent_videos).to eq([gotham, monk])
+		end
+
+		it "returns all videos if there are less than 6" do
```

Comment:
This is a duplicate spec of above but it does not meet the criteria of the description. It does not return all videos, only the videos in the given category.